syft/pkg/cataloger/binary/cataloger.go

```diff
@@ -0,0 +1,78 @@
+package binary
+
+import (
+	"github.com/anchore/syft/syft/artifact"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
+)
+
+const catalogerName = "binary-cataloger"
+
+func NewBinaryCataloger() pkg.Cataloger {
+	return &binaryCataloger{}
+}
+
+// binaryCataloger is the cataloger responsible for surfacing evidence of a very limited set of binary files,
+// which have been identified by the classifiers. The binaryCataloger is _NOT_ a place to catalog any and every
+// binary, but rather the specific set that has been curated to be important, predominantly related to toolchain-
+// related runtimes like Python, Go, Java, or Node. Some exceptions can be made for widely-used binaries such
+// as busybox.
+type binaryCataloger struct{}
```

Comment:
```suggestion
type Cataloger struct{}
```
same reason as https://github.com/anchore/syft/pull/1336/files#r1035187073